style.css

```diff
@@ -1,32 +1,44 @@
 * {
- margin: 0;
- padding: 0;
- width: 100%;
- box-sizing: border-box;
- border-style: 1px solid black;
+  margin: 0;
+  padding: 0;
+  height: 100%;
```

Comment:
you don't have to take it out here, but both height and width of 100% shouldn't be in the * selector